src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/utils/DataHelpers.kt

```diff
@@ -14,10 +14,28 @@ class DataHelpers {
   companion object {
 
     fun makeVideoLinkBooking(id: Long): VideoLinkBooking =
```

Comment:
we don't seem to be using this in many places - would be nice to refactor tests to make more usage of it